bin/cluster_sce.R

```diff
@@ -65,30 +65,35 @@ if (!file.exists(opt$processed_sce_file)) {
 }
 sce <- readr::read_rds(opt$processed_sce_file)
 
+# only perform clustering if reduced dimension embeddings are present
+# otherwise just return the object
+if(length(reducedDimNames(sce)) > 0){
 
```

Comment:
I think it is probably worth warning here, just for log purposes, so I would flip this around:
```suggestion
if(length(reducedDimNames(sce)) == 0) {
  warning("No reduced dimensions present, skipping clustering")
} else {
```